src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/_help.py

```diff
@@ -9,5 +9,5 @@
 
 helps['appservice'] = """
     type: group
-    short-summary: Create and deploy web apps
+    short-summary: commands for managing your azure web apps, app service plans, etc
```

Comment:
Reasonable suggestion. I just copied and pasted the proposed text from webapp team :-).
I will get Azure capitalized at least. 
The `etc` doesn't include anything more yet, but sub-features will get in and people are welcome to explorer.